src/components/PhnInput.vue

```diff
@@ -88,58 +91,52 @@ export const phnValidator = (value) => {
 };
 
 export default {
-  name: 'PhnInput',
+  name: "PhnInput",
   directives: { maska },
-  mixins: [
-    blurMixin,
-    cypressMixin,
-  ],
+  mixins: [blurMixin, cypressMixin],
```

Comment:
Strange, I think the eslint rulesets aren't applying the same for us. Looks like it is undoing the `array-element-newline` rule